tarballs.json

```diff
@@ -1 +1 @@
-{"libpysal": ["v4.0.1", "https://api.github.com/repos/pysal/libpysal/tarball/v4.0.1"], "esda": ["v2.0.1", "https://api.github.com/repos/pysal/esda/tarball/v2.0.1"], "giddy": ["v2.0.0", "https://api.github.com/repos/pysal/giddy/tarball/v2.0.0"], "inequality": ["v1.0.0", "https://api.github.com/repos/pysal/inequality/tarball/v1.0.0"], "pointpats": ["V1.1.0", "https://api.github.com/repos/pysal/pointpats/tarball/V1.1.0"], "spaghetti": ["1.1.1.post3", "https://api.github.com/repos/pysal/spaghetti/tarball/1.1.1.post3"], "mapclassify": ["v2.0.1", "https://api.github.com/repos/pysal/mapclassify/tarball/v2.0.1"], "spreg": ["v1.0.4post1", "https://api.github.com/repos/pysal/spreg/tarball/v1.0.4post1"], "spglm": ["v1.0.6", "https://api.github.com/repos/pysal/spglm/tarball/v1.0.6"], "spint": ["v1.0.4", "https://api.github.com/repos/pysal/spint/tarball/v1.0.4"], "mgwr": ["v2.0.1", "https://api.github.com/repos/pysal/mgwr/tarball/v2.0.1"], "spvcm": ["0.2.0", "https://api.github.com/repos/pysal/spvcm/tarball/0.2.0"]}
\ No newline at end of file
+{"libpysal": ["v4.0.1", "https://api.github.com/repos/pysal/libpysal/tarball/v4.0.1"], "esda": ["v2.0.1", "https://api.github.com/repos/pysal/esda/tarball/v2.0.1"], "giddy": ["v2.1.0", "https://api.github.com/repos/pysal/giddy/tarball/v2.1.0"], "inequality": ["v1.0.0", "https://api.github.com/repos/pysal/inequality/tarball/v1.0.0"], "pointpats": ["v2.0.0", "https://api.github.com/repos/pysal/pointpats/tarball/v2.0.0"], "spaghetti": ["v1.2", "https://api.github.com/repos/pysal/spaghetti/tarball/v1.2"], "mapclassify": ["v2.0.1", "https://api.github.com/repos/pysal/mapclassify/tarball/v2.0.1"], "splot": ["v1.0.0", "https://api.github.com/repos/pysal/splot/tarball/v1.0.0"], "spreg": ["v1.0.4post1", "https://api.github.com/repos/pysal/spreg/tarball/v1.0.4post1"], "spglm": ["v1.0.6", "https://api.github.com/repos/pysal/spglm/tarball/v1.0.6"], "spint": ["v1.0.5", "https://api.github.com/repos/pysal/spint/tarball/v1.0.5"], "mgwr": ["v2.0.2", "https://api.github.com/repos/pysal/mgwr/tarball/v2.0.2"], "spvcm": ["v0.2.1post1", "https://api.github.com/repos/pysal/spvcm/tarball/v0.2.1post1"]}
```

Comment:
```python
"spaghetti": ["v1.2", "https://api.github.com/repos/pysal/spaghetti/tarball/v1.2"]
```
should be
```python
"spaghetti": ["v1.3", "https://api.github.com/repos/pysal/spaghetti/tarball/v1.3"]
```
correct?

Reply:
ah, nice catch James. segregation will need to be updated too for the release we are doing on Monday. @renanxcortes and I are thinking we'll do that as 1.1.0

Reply:
Good catch indeed.

 I forgot to add the newest version of that file to the commit. That one has the correct versions for the packages.

I'm planning on giving a short overview of the build process at the july dev meeting so others can see how to update the info for the upstream packages for the meta build.